protos/remote_id/remote_id.proto

```diff
@@ -0,0 +1,125 @@
+}
+message SetLocationResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message SetSystemRequest {
+    SystemId system = 1; // Desidered system
+}
+message SetSystemResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message SetOperatorIdRequest {
+    OperatorId system = 1; // Desidered operator_id
+}
+message SetOperatorIdResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message SetSelfIdRequest {
+    SelfId self_id = 1; // Desidered self_id
+}
+message SetSelfIdResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message BasicId {
+    uint32 id_type = 1; // Indicates the format for the uas_id field of this message.
+    uint32 ua_type = 2; // Indicates the type of UA (Unmanned Aircraft).
+    string uas_id = 3; // UAS ID following the format specified by id_type.
+}
+
+message Location {
+    uint32 status = 1; // Indicates whether the unmanned aircraft is on the ground or in the air.
+    uint32 direction_deg = 2; // Direction over ground measured clockwise from true North: 0 - 359 deg.
+    uint32 speed_horizontal_cm_s = 3; // Ground speed. Positive only. cm/s
+    int32 speed_vertical_cm_s = 4; // The vertical speed. Up is positive. cm/s
+    float latitude_deg = 5; // Current latitude of the unmanned aircraft
+    float longitude_deg = 6; // Current longitude of the unmanned aircraft
+    float altitude_barometric_m = 7; // The altitude calculated from the barometric pressure.
+    float altitude_geodetic_m = 8; // The geodetic altitude as defined by WGS84.
+    uint32 height_reference = 9; // Indicates the reference point for the height field.
+    float height_m = 10; // The current height of the unmanned aircraft. As indicated by height_reference.
+    float timestamp_s = 11; // Seconds after the full hour with reference to UTC time.
+    uint32 timestamp_accuracy = 12; // The accuracy of the timestamps.
+}
+
+message SystemId {
+    uint32 operator_location_type = 1; // Specifies the operator location type.
+    uint32 classification_type = 2; // Specifies the classification type of the UA.
+    float operator_latitude_deg = 3; // Latitude of the operator.
+    float operator_longitude_deg = 4; // Longitude of the operator.
+    uint32 area_count = 5; // Number of aircraft in the area,
+    uint32 area_radius_m = 6; // Radius of the cylindrical area of the group or formation.
+    float area_ceiling_m = 7; // Area Operations Ceiling relative to WGS84.
+    float area_floor_m = 8; // Area Operations Floor relative to WGS84.
+    uint32 category_eu = 9; // When classification_type is MAV_ODID_CLASSIFICATION_TYPE_EU, specifies the category of the UA.
+    uint32 class_eu = 10; // When classification_type is MAV_ODID_CLASSIFICATION_TYPE_EU, specifies the class of the UA.
+    float operator_altitude_geo_m = 11; // Geodetic altitude of the operator relative to WGS84. If unknown: -1000 m.
+    uint32 timestamp_s = 12; // s32 bit Unix Timestamp in seconds since 00:00:00 01/01/2019.
```

Comment:
Wow, they came up with some weird types in the opendrone ID MAVLink messages. A UNIX timestamp but starting in 2019. It'll overflow in year 2155. Well, won't be my problem :joy:

Reply:
I wonder if we want to expose this or if MAVSDK should read the time from the system and avoid having the user to deal with it.

Reply:
> if MAVSDK should read the time from the system

I agree it would be nice, but I'm scared of it. MAVSDK runs on different platforms, and reading the time may not be as straightforward as we may think? Just a thought, I don't have a strong opinion there.

Reply:
Yes, that's potentially an issue. Also local time or UTC? :thinking:

Reply:
The time is UTC but:
I'm experimenting with the plugin and I tried to fill the timestamp for the user but the problem I see is that a  reliable clock source in a embedded system, maybe, is a requirement too strong for the library.
Instead the precision of clock is important because the remoteid standard said that packets must be rejected if they are more old than one second. 
So in the end I think that is better let user get the responsibility to deal with timestamps.

Reply:
Maybe, We can let user provide a valid UTC timestamp and let plugin make the weird conversions?

Reply:
I'd say so